server/tests/audit_math/snapshots/snap_test_sampler.py

```diff
@@ -20,6 +33,8 @@
     ("0.230651143", "pct 7", 2),
 ]
 
+snapshots["test_draw_macro_sample_wrong_contest 1"] = []
```

Comment:
where does this come from? i can't find a test case that corresponds

Reply:
hmm, that's weird. Lemme regen the snapshots for this branch.